requirements.in

```diff
@@ -4,4 +4,4 @@
 jinja2
 ops
 lightkube
-serialized-data-interface<0.4
+serialized-data-interface==0.3.6
```

Comment:
I'd leave it as <0.4, it will anyway pull the version we want. Also, please add a note:

```
# dex-auth's current implementation is not compatible
# with SDI>0.3.
```

Reply:
Will do.